src/musicbrainz/musicbrainzclient.cpp

```diff
@@ -73,24 +72,19 @@ void MusicBrainzClient::requestFinished() {
     ResultList ret;
 
     int status = reply->attribute(QNetworkRequest::HttpStatusCodeAttribute).toInt();
+    qDebug() << "MusicBrainzClient GET reply status:" << status;
 
     // MusicBrainz returns 404 when the MBID is not in their database. We treat
     // a status of 404 the same as a 200 but it will produce an empty list of
     // results.
     if (status != 200 && status != 404) {
-        QTextStream body(reply);
-        qDebug() << "MusicBrainzClient GET reply status:" << status << "body:" << body.readAll();
```

Comment:
Since we were having trouble with MusicBrainz/AcoustId network requests and it's a rarely used feature I thin it's useful to print the body. Could you keep that?